packages/contracts-bedrock/src/L1/L1StandardBridge.sol

```diff
@@ -211,7 +211,9 @@ contract L1StandardBridge is StandardBridge, ISemver {
     /// @param _minGasLimit Minimum gas limit for the deposit message on L2.
     /// @param _extraData   Optional data to forward to L2.
     function _initiateETHDeposit(address _from, address _to, uint32 _minGasLimit, bytes memory _extraData) internal {
-        _initiateBridgeETH(_from, _to, msg.value, _minGasLimit, _extraData);
+        _initiateBridgeETHToERC20(
+            0x4200000000000000000000000000000000001023, _from, _to, msg.value, _minGasLimit, _extraData
```

Comment:
Maybe set this as a constant with a name? But I don't totally know if that could cost or not more gas to be honest

Reply:
Ahh yes. When I started out I was not sure were to put the named constant, and later I just forgot. Fixed now.